File: app/src/main/java/com/navana/zabaandemo/MyApplication.java
package com.navana.zabaandemo;

import android.app.Application;

import com.zabaan.sdk.InitializeParams;
import com.zabaan.sdk.Zabaan;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        String token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJmcmVzaCI6ZmFsc2UsImlhdCI6MTYyNzYyMjA0NiwianRpIjoiNGJiNzU2MzQtMDE1NS00ODI0LTg0MTAtODk1YjMxMTA2NzRiIiwidHlwZSI6ImFjY2VzcyIsInN1YiI6IjE3NzBhMTJmLWM3ODItNDlmZi1iMDNiLWZmNTM0OTEwODEzNyIsIm5iZiI6MTYyNzYyMjA0NiwiZGVwbG95bWVudF9pZCI6IjkwMmUwNWI1LWMzODMtNGNiYi1hZjQzLTc0MTQ5NmRjYzk0MCIsInByb2plY3RfaWQiOiJhYWY2YWUwNi1hMTFkLTQwM2YtOTljNC0zY2FiY2M5MThjN2YiLCJyb2xlIjoic2RrX3VzZXIifQ.vIgHRMDwdwmsQj915SaFN4oNJW7y-nfxqmz8891RXUqFxuTsejer9N4g6r8mRkDS3EY0uD46PAdmHtXvoDc3OQ";
        try {
            InitializeParams params = new InitializeParams.Builder()
                    .context(this)
                    .accessToken(token)
                    .build();
            Zabaan.setDebug(true);
            Zabaan.initialize(params);
        } catch (Exception e) {
            //Handle the exception here.
        }
    }
}

File: app/src/main/java/com/demoapp/hdfcdemo/MyApplication.kt
package com.demoapp.hdfcdemo

import android.content.Context
import android.util.Log
import androidx.multidex.MultiDexApplication
import com.demoapp.hdfcdemo.MyApplication
import com.zabaan.sdk.InitializeParams
import com.zabaan.sdk.Zabaan
import java.lang.Exception

class MyApplication : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        appContext = applicationContext
        try {
            val params: InitializeParams = InitializeParams.Builder()
                .context(this)
                .accessToken(BuildConfig.ACCESS_TOKEN)
                .build()
            Zabaan.setDebug(true)
            Zabaan.setSandbox(true)
            Zabaan.init(params)
        } catch (e: Exception) {
            Log.e("Exception", "Navana Assistant Failed to initialize : "+e.message)
        }
    }

    override fun onTerminate() {
        super.onTerminate()
    }

    companion object {
        var appContext: Context? = null
            private set
        var isIntroPlayed = false
    }
}